music/src/core/data.ts

```diff
@@ -460,7 +466,6 @@ abstract class MelodyControlConverter extends DataConverter {
  * `toNoteSequence` returns a `NoteSequence` with drum hits at the note-on
  * steps.
  */
-export interface MelodyRhythmConverterArgs extends BaseConverterArgs {}
```

Comment:
why remove these?

Reply:
It was actually never being used, the MelodyRhythmConverter just uses the MelodyConverterArgs.